app/controllers/core_induction_programmes/modules_controller.rb

```diff
@@ -33,6 +60,18 @@ def load_course_module
   end
 
   def course_module_params
-    params.require(:course_module).permit(:content, :title, :term)
+    params.require(:course_module).permit(:content, :title, :term, :course_year_id)
+  end
+
+  def find_course_year
+    CourseYear.find_by(id: params[:course_module][:course_year])
+  end
+
+  def set_previous_module(previous_module)
+    CourseModule.where(id: previous_module).pick(:previous_module_id)
+  end
```

Comment:
So if we have module `A`, and another called `C`, and we insert `B` into the mix, what do we do?

We select `A` as previous module. But in here, we select the module _before_ A, which doesn't exist. So do we end up with B-A-C order?

Reply:
So the user selects the module they want to follow the one they're creating. We're not selecting A as the previous for example, it's that A will follow it. I started off by selecting the previous module but changed because the one you want to follow has the existing previous module ID and will be the one you want to assign your new module to. Also it means in the select we don't have to include an additional option of say "Put this number one". It can be number one by selecting the first module. On the collection_select i've got a hint that says `"hint: { text: "The module you are creating will precede the module you are selecting" }`